weblate/accounts/notifications.py

```diff
@@ -441,9 +441,15 @@ def notify_immediate(self, change) -> None:
                 if current_subscription.onetime:
                     current_subscription.delete()
 
-    def send_digest(self, language, email, changes, subscription=None) -> None:
+    def send_digest(
+        self, language, email, changes, subscription=None, overlimit=False
```

Comment:
Please add type annotations to the new code:

```suggestion
        self, language, email, changes, subscription=None, *, overlimit: bool = False
```

Making boolean kwargs only avoids mistakes in invocation.